### ISSUE TITLE
Compatability with OCaml 5

### ISSUE THREAD

**Issue by mgttlinger** [tcsprojects/pgsolver#30]
Are there any plans to support OCaml 5?

**Comment by oliverfriedmann** [comment]
Yes, but with low priority. Without having tested it - shouldn't most of the code work "out of the box" with OCaml 5?

**Comment by mgttlinger** [comment]
Yeah. I think last time I checked the trouble was mostly that oasis didn't compile with OCaml 5 due to some dependency not being compatible.
